pkg/v1/tkg/test/data/testdata/customClusterBootstrapAntrea.yaml

```diff
@@ -22,7 +22,7 @@ apiVersion: run.tanzu.vmware.com/v1alpha3
 kind: ClusterBootstrap
 metadata:
   annotations:
-    tkg.tanzu.vmware.com/add-missing-fields-from-tkr: v1.23.5---vmware.1-tkg.1-zshippable
+    tkg.tanzu.vmware.com/add-missing-fields-from-tkr: v1.22.11---vmware.2-tkg.1-zshippable
```

Comment:
Changes to this file are only for local testing of this pr. Eventually they should be reverted before merging. No need to make this patch larger than it needs to be.